index.html

```diff
@@ -41,14 +44,17 @@
             <div class="main-navigation">
                 <div class="logo">
                     <a href="index.html">
-                        <img src="">
+                        <img alt="Robust-Events" src="img/logo.png" class="main-logo">
                     </a>
                 </div>
 
                 <div class="menu-icon"></div>
```

Comment:
- [ ] Please, consider fixing the mobile menu bottom side: it is not cover the whole page 👍 
![2022-03-09 22 24 27](https://user-images.githubusercontent.com/49064106/157507264-b727dddf-a520-415f-9750-f5b790d7bc39.gif)